draft/spec/index.html

```diff
@@ -1514,6 +1547,27 @@ <h2>Moab in an OCFL Object</h2>
 }
 </pre>
 </section>
+    <section id="example-extended-object">
```

Comment:
Perhaps we need an example Storage Root layout to clarify the extensions mechanism there?

Reply:
:+1: